relay/server_test.go

```diff
@@ -30,6 +30,7 @@ func defaultConfig() *Config {
 		ChunkCacheSize:             1024 * 1024,
 		ChunkMaxConcurrency:        32,
 		MaxKeysPerGetChunksRequest: 1024,
+		AuthenticationDisabled:     true,
```

Comment:
if we set this to false, does integration test fail?

Reply:
Disabling authentication doesn't cause signed requests to be rejected, it just causes signatures to be entirely ignored by the relay.

That being said, disabling authentication in the relay tests was a bit of a shortcut (the bad kind). I've modified the code in this test file to properly enable authentication.